src/core/actions/specs/action-in-workflow.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { Cli } from "@wbce/services";
+import { Action } from "../src/action-manager";
+import { ActionState } from "../src/models/action";
+import { Executor } from "../src/action-executor";
+import { TestActionInWorkflow, TestExecutorAction } from "./test-action";
+
+
+describe("Test action defined in workflow", ()=>{
```

Comment:
Would be nice to have a .md at the root of the spec folder to specify how to launch the test suite.